pcdsdevices/epics_motor.py

```diff
@@ -376,6 +376,13 @@ def flag_is_cleared(value=None, **kwargs):
             status_wait(st, timeout=timeout)
         return st
 
+    def camonitor(self):
```

Comment:
I wonder if this should become a standard method on every `Signal` object...

Reply:
@teddyrendahl are you opposed to awful monkeypatching? E.g. if we defined a `camonitor` method standalone and did:
```
from ophyd.signal import Signal

def camonitor(self):
    <code>

Signal.camonitor = camonitor
```
Then we could use this in literally every class as desired. I'm not sure what Dan and Tom would think about putting this as part of `ophyd`, but it seems like a reasonable interactive debugging tool.